package.json

```diff
@@ -18,6 +18,8 @@
     "MediaWiki"
   ],
   "author": "Wikimedia Service Team <services@wikimedia.org>",
+  "contributors": [
```

Comment:
Yes, so that devs know its available and start appending to it from the start. Without it most contributors wouldn't know they can do it.